src/AvatarList/index.js

```diff
@@ -32,14 +33,21 @@ class AvatarList extends Component {
   }
 
   render() {
-    const { imageList, imageSpacing, imageChild, _fonts } = this.props
+    const { imageList, imageSpacing, imageChild, _fonts, listEmptyState } =
+      this.props
 
     if (
       !imageList ||
       typeof navigator.userAgent === undefined ||
       !imageList[0]
     ) {
-      return <View style={{ height: imageSize }}></View>
+      //return <View style={{ height: imageSize }}></View>
```

Comment:
This is a check for if the list hasn't loaded yet. Still just return an empty view, no need to worry about empty state here.

Reply:
On second thought, we may just want to split this logic out a little. If `imageList` is an empty array, that's when you want to render the empty state.